src/LayerPlan.h

```diff
@@ -334,17 +334,40 @@ class LayerPlan : public NoCopy
 
     const Polygons* getCombBoundaryInside() const
     {
-        return &comb_boundary_inside2;
+        return &comb_boundary_preferred;
     }
 
 private:
     /*!
-     * \brief Compute the boundary within which to comb, or to move into when
-     * performing a retraction.
-     * \param max_inset The greatest inset index.
-     * \return the comb_boundary_inside
+     * Determines if the combing boundary need to be computed
+     * \return true if there should be a combing designated surface on this layer, otherwise false.
      */
-    Polygons computeCombBoundaryInside(const size_t max_inset);
+    bool CombBoundaryRequired();
+
+    /*!
+     * Compute the minimum combing boundary.
+     *
+     * This is in most cases within half a wall line width smaller then the part outlines. Except for raft layers.
```

Comment:
Currently the minimum combing boundary is a full Outer Wall Line Width smaller than the outlines. Not half.

This might change if you implement my earlier concern that this area is wrong though.